src/tribler/core/components/database/db/tribler_database.py

```diff
@@ -1,14 +1,20 @@
 import logging
+import os
 from typing import Any, Optional
 
 from pony import orm
 
 from tribler.core.components.database.db.layers.health_data_access_layer import HealthDataAccessLayer
 from tribler.core.components.database.db.layers.knowledge_data_access_layer import KnowledgeDataAccessLayer
-from tribler.core.utilities.pony_utils import TrackedDatabase, get_or_create
+from tribler.core.utilities.pony_utils import TrackedDatabase, db_session, get_or_create
+
+MEMORY = ':memory:'
 
 
 class TriblerDatabase:
+    CURRENT_VERSION = 1
+    _SCHEME_VERSION = 'scheme_version'
```

Comment:
**nitpick:** The `_version` suffix is a bit confusing here, as the variable does not contain the version value. Can we rename it to something like `version_key`?

Reply:
Renamed